docs/basics/UsageWithReact.md

```diff
@@ -74,196 +72,281 @@ I see the following components (and their props) emerge from this brief:
 * **`Footer`** is a component where we let user change visible todo filter.
   - `filter: string` is the current filter: `'SHOW_ALL'`, `'SHOW_COMPLETED'` or `'SHOW_ACTIVE'`.
   - `onFilterChange(nextFilter: string)`: Callback to invoke when user chooses a different filter.
+* **`Link`** is a link with a callback.
+  - `onClick()` is a callback to invoke when link is clicked.
 
 These are all presentational components. They don’t know *where* the data comes from, or *how* to change it. They only render what’s given to them.
 
 If you migrate from Redux to something else, you’ll be able to keep all these components exactly the same. They have no dependency on Redux.
 
-Let’s write them! We don’t need to think about binding to Redux yet. You can just give them fake data while you experiment until they render correctly.
+We also have some container components that connects to Redux. The container components are responsible for getting and transforming state. Remember that changes result in a new state as state is immutable.
```

Comment:
I think "getting and transforming state" is a bit confusing. Changes happen in reducer, so I'm not sure why the next sentence is relevant here.

Maybe something like "Container components calculate the props to pass to the presentational components they wrap based on the current state of the Redux store." could work better instead.